### PR TITLE
Serde patch

### PR DESCRIPTION
**Pull request by tudorcebere**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: src/syft/core/pointer/pointer.py [hunk 1]
```diff
@@ -425,9 +425,6 @@ def request(
 
         self.client.send_immediate_msg_without_reply(msg=msg)
 
-        # wait long enough for it to arrive and trigger a handler
-        time.sleep(0.1)
```
Comment (madhavajay) [inline src/syft/core/pointer/pointer.py:429]: This might actually make things slower if you are using the request_handlers.

**Comment by madhavajay** [comment]
@tudorcebere This looks like it breaks the Grid unit tests. We could hold this one off until we go monorepo, or you could checkout the PyGrid repo into the root syft dir and then run the tests the same way CI does and make a PR for Grid that brings them together.

**Review by madhavajay — CHANGES_REQUESTED** [review]
This breaks PyGrid Unit tests, so we should make sure theres a PR on the other side which integrates these changes.

**Comment by tudorcebere** [comment]
I'm fine with mono repo first, then this change.

**Comment by madhavajay** [comment]
@tudorcebere can we update this with the mono repo and get it merged before there are too many new post `0.5.0` changes.

**Comment by madhavajay** [comment]
@tudorcebere I definitely still want to see this PR. Lets discuss when you have cycles again.

**Pull request closed without merging**